user/src/main/java/com/user/dtoConversion/DtoConversion.java

```diff
@@ -0,0 +1,50 @@
+package com.user.dtoConversion;
+
+import com.user.entity.Address;
+import com.user.entity.User;
+import com.user.inDto.AddressRequest;
+import com.user.inDto.UserInDto;
+import com.user.outDto.AddressResponse;
+import com.user.outDto.UserOutDto;
+
+public class DtoConversion {
+  public static User requestToUser(UserInDto userInDto) {
```

Comment:
If you are going to use the same class for all dto conversions then method name should be self explanatory, like the name should be userInDtoToUser